core/controllers/suggestion_test.py

```diff
@@ -1641,59 +1641,6 @@ def test_suggestion_creation_when_images_are_not_provided(self):
             response_dict['error'])
         self.logout()
 
-    def test_suggestion_creation_when_images_are_not_valid(self):
```

Comment:
Why is this test removed?

Reply:
Initial test was added for question suggestion. Since images are sent before the question is submitted this is not happening actually and I added this test with as translation suggestion.

Reply:
Must images be stored to the backend before the question is submitted? What happens if they're stored and then the question edit is cancelled -- will those images still remain in the backend?

If so can things be modified such that that is not the case?

Reply:
1. Must images be stored to the backend before the question is submitted? **Yes in this case.**

2. What happens if they're stored and then the question edit is cancelled -- will those images still remain in the backend? **As it stands, images will still remain in the backend if the question edit is cancelled.**

3. If so can things be modified such that that is not the case? **As things stand in `fs_services.py` I can not find a function to remove saved images.**

Reply:
Can we not save the images in the backend until the "submit question" button is pressed? (And, until then, can they just be saved in local storage in the frontend?)

Reply:
Yes, I think that's the way to go but I am blocked while implementing as I said here https://github.com/oppia/oppia/pull/14289#discussion_r754144355

Reply:
@chris7716 Do you still need anything from me for this comment?

Reply:
No. What you said in the offline discussion is enough and I think we are going to implement this in the Stage 2. Thanks.